mailqueue/utils.py

```diff
@@ -27,4 +27,4 @@ def upload_to(instance, filename):
     filename = filename.split('/')[-1]
     # Because instead of filesystem, email message
     # can have multiple attachments with the same filename
-    return 'mailqueue-attahcments/{0}_{1}'.format(get_random_string(length=24), filename)
+    return 'mailqueue-attachments/{0}_{1}'.format(get_random_string(length=24), filename)
```

Comment:
I'm concerned that changing this now would break existing file locations.  Typos are annoying, but it doesn't really solve problem without creating a much bigger one, correct?